evm-adapters/testdata/DebugLogs.sol

```diff
@@ -15,6 +15,7 @@ contract DebugLogs is DSTest {
         emit log_named_address("addr", 0x2222222222222222222222222222222222222222);
         emit log_named_bytes32("key", keccak256(abi.encodePacked("foo")));
         emit log_named_decimal_int("key", 123, 18);
+        emit log_named_decimal_int("key", -123, 18);
         emit log_named_decimal_uint("key", 1234, 18);
```

Comment:
Unrelated, but do we have a bug here? Should the 1234 be expanded to 1234 & 18 0s, similar to the int counterparts?

Reply:
I'll take a look

Reply:
To be clear, the difference is:
```
/// Multiplies the provided amount with 10^{units} provided.
pub fn parse_units<K, S>(amount: S, units: K) -> Result<U256, ConversionError>
```
```
/// Divides the provided amount with 10^{units} provided.
pub fn format_units<T, K>(amount: T, units: K) -> Result<String, ConversionError>
```

So we're doing `U256 <-> String`. 

These should both multiply, right?

Reply:
I'll create a new issue and discussion is needed